### ISSUE TITLE
训练损失一直下不去

### ISSUE THREAD

**Issue by HUST-WangZhimin** [Megvii-BaseDetection/YOLOX#284]
作者你好！我用你们的yolox训练自己的数据集时发现total_loss一直震荡下不去，我使用的数据集一共有3000张图片，训练集验证集的划分比例为8：2。请问需要进行哪些改动才可以让total_loss降下去呢？谢谢！
![1](https://user-images.githubusercontent.com/53384442/127726135-9482046d-ee64-4750-beba-1517e457b9a7.png)
![2](https://user-images.githubusercontent.com/53384442/127726151-e0844fed-5eb7-4b2a-b622-28f7dcddb23d.png)

**Comment by scuizhibin** [comment]
我的一直在4.4左右徘徊，我使用的是voc数据集和。

**Comment by Joker316701882** [comment]
@HUST-WangZhimin How is your evaluation mAP?

**Comment by scuizhibin** [comment]
map5095&nbsp;&nbsp;57.6&nbsp;&nbsp;map50&nbsp;&nbsp;83。这个怎么样？voc数据集和上训练的。


---原始邮件---
发件人: &quot;Ge ***@***.***&gt;
发送时间: 2021年7月31日(周六) 下午5:14
收件人: ***@***.***&gt;;
抄送: ***@***.******@***.***&gt;;
主题: Re: [Megvii-BaseDetection/YOLOX] 训练损失一直下不去 (#284)




 
@HUST-WangZhimin How is your evaluation mAP?
 
&mdash;

**Comment by Joker316701882** [comment]
@scuizhibin This is normal performance.

**Comment by HUST-WangZhimin** [comment]
> @HUST-WangZhimin How is your evaluation mAP?

The evaluation mAP is around 0.83，I trained 100 epochs

**Comment by scuizhibin** [comment]
map&nbsp;&nbsp;0.83


---Original---
From: ***@***.***&gt;
Date: Mon, Aug 2, 2021 17:24 PM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [Megvii-BaseDetection/YOLOX] High training loss (#284)




  
@HUST-WangZhimin How is your evaluation mAP?
  
The evaluation mAP is around 0.83，I trained 100 epochs
 
&mdash;

**Comment by scuizhibin** [comment]
我加入了注意力结果也才有0.84，最小的模型。


---Original---
From: ***@***.***&gt;
Date: Mon, Aug 2, 2021 17:24 PM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [Megvii-BaseDetection/YOLOX] High training loss (#284)




  
@HUST-WangZhimin How is your evaluation mAP?
  
The evaluation mAP is around 0.83，I trained 100 epochs
 
&mdash;

**Comment by scuizhibin** [comment]
0.84map在voc数据上，我加入了注意力机制，提升的不明显啊


---原始邮件---
发件人: &quot;Ge ***@***.***&gt;
发送时间: 2021年7月31日(周六) 下午5:21
收件人: ***@***.***&gt;;
抄送: ***@***.******@***.***&gt;;
主题: Re: [Megvii-BaseDetection/YOLOX] 训练损失一直下不去 (#284)




 
@scuizhibin This is normal performance.
 
&mdash;

**Comment by charles-str** [comment]
> 作者你好！我用你们的yolox训练自己的数据集时发现total_loss一直震荡下不去，我使用的数据集一共有3000张图片，训练集验证集的划分比例为8：2。请问需要进行哪些改动才可以让total_loss降下去呢？谢谢！ ![1](https://user-images.githubusercontent.com/53384442/127726135-9482046d-ee64-4750-beba-1517e457b9a7.png) ![2](https://user-images.githubusercontent.com/53384442/127726151-e0844fed-5eb7-4b2a-b622-28f7dcddb23d.png)

您训练的是yolox-tiny吗？交流下
